src/main/resources/update/gitbucket-core_4.35.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<changeSet>
+  <!--================================================================================================-->
+  <!-- ACCOUNT_HIGHLIGHTER -->
+  <!--================================================================================================-->
+  <createTable tableName="ACCOUNT_HIGHLIGHTER">
```

Comment:
If we will have per-user preferences in GitBucket, it might be better to define a table like `ACCOUNT_PREFERENCES` or `USER_PREFERENCES` so that we can add columns for another per-user preference items to this table later.